src/js/components/InfiniteScroll/index.d.ts

```diff
@@ -1,12 +1,12 @@
-import * as React from "react";
+import * as React from 'react';
 
 export interface InfiniteScrollProps {
-  children?: ((...args: any[]) => any);
-  items?: any[];
-  onMore?: ((...args: any[]) => any);
-  renderMarker?: ((...args: any[]) => any);
+  children?: (...args: [string, number]) => React.ReactElement;
```

Comment:
Is the number optional?

Reply:
I'm actually going to change children back to any now. I think that `children` should technically match the types for `items` since infinite scroll. however, copying the types from line 5 into the children args cause an error for the `key` prop which was looking for a string or number. I think for now it is safest to leave the children arg as `any`

Reply:
Can we remove `children` and just fall back to the default children, or do we need it because of the function?

Reply:
Good call. Let me look into that.

Reply:
It seems we can remove it, this is behaving similar to how children is used in Menu or Button, and we don't define types there. Updating PR now.